### DISCUSSION TITLE
Identifier uniquenes violation 에러 질문

### DISCUSSION THREAD

**Discussion by hemil0102** [hemil0102/XRStudy#41] [7. Q&A]
유니티에서 제공하는 AR_Marker_Assets.unitypackgage를 다운 받아서 Assets 폴더에 넣고 Import를 하였는데, 
스샷과 같은 에러가 여러개 발생했습니다. 어떻게 없앨 수 있을까요? 
![Screenshot 2024-06-27 at 11 29 31 PM](https://github.com/hemil0102/XRStudy/assets/83139316/aa3eac05-15d1-4eb3-95b4-61bc9ca5a451)

**Comment by jongfeel** [comment]
저도 main branch 기준으로 프로젝트 열어보고 확인해 보겠습니다.

**Comment by jongfeel** [comment]
Import 할 때 경고 문제인 것 같은데 실행할 때 문제 없고 다시 warning이 뜨지 않는다면 상관 없을 것 같습니다.
